include/deal.II/base/table_indices.h

```diff
@@ -140,22 +140,30 @@ TableIndices<N>::TableIndices(const unsigned int index0,
 
   switch (N)
     {
-    case 1: // fallthrough
+    case 1:
       Assert (index1 == numbers::invalid_unsigned_int, ExcMessage("more than N index values provided"));
-    case 2: // fallthrough
+      DEAL_II_FALLTHROUGH
```

Comment:
Fine with me. At least `astyle` seems to already handle this correctly in either form.